src/PlotlyLight.jl

```diff
@@ -252,7 +255,13 @@ function page(o::Plot)
     ))
 end
 
-Base.display(::Cobweb.CobwebDisplay, o::Plot) = display(Cobweb.CobwebDisplay(), page(o))
+function Base.display(::Cobweb.CobwebDisplay, o::Plot) 
+    if (~isinteractive() && isdefined(Main, :VSCodeServer)) # VSCode Jupyter
+        display(HTML(repr("text/html", o)));
+    else
+        display(Cobweb.CobwebDisplay(), page(o))
+    end
+end
```

Comment:
I don't think this is the right approach.  You're using the `display(::CobwebDisplay, x)` method to possibly not use that display.

Reply:
Would it make sense to do something like this then?
```
function Base.display(::Cobweb.CobwebDisplay, o::Plot) 
    if (~isinteractive() && isdefined(Main, :VSCodeServer)) # VSCode Jupyter
        display(Cobweb.CobwebDisplay(), HTML(repr("text/html", o)))
    else
        display(Cobweb.CobwebDisplay(), page(o))
    end
end
```

the 1st `HTML(repr("text/html", o))` option actually works in VSCode REPL as well but not in standalone Julia REPL.

Or do you have a better suggestion?

Reply:
resolved by setting display_object to Cobweb.IFrame for when jupyter is running in vscode